python/tank/bootstrap/cached_configuration.py

```diff
@@ -276,6 +289,23 @@ def update_configuration(self):
                     self.path.as_shotgun_dict()
                 )
 
+    def cleanup_backup_folders(self):
+        """
+        Cleans up backup folders generated by a call to the update_configuration method
+        """
+        if self.config_backup_folder_path is not None:
+            try:
+                filesystem.delete_folder(self.config_backup_folder_path)
+                self.config_backup_folder_path = None
+            except Exception, e:
+                log.error("Failed to clean up the previous installation folder at: %s" % self.config_backup_folder_path)
```

Comment:
please include the exception message in the logged error - it's helpful to know *why* it failed, not just that it failed.